_topic_map.yml

```diff
@@ -138,6 +138,8 @@ Topics:
   #   File: installing-openstack-installer-kuryr
   # - Name: Load balancing deployments on OpenStack
   #   File: installing-openstack-load-balancing
+  - Name: Uninstalling a cluster on OpenStack
+    File: uninstalling-cluster-osp
```

Comment:
Why's the initialism creeping into your file name?

Reply:
Fiiiiine.